plugins/plugin-chart-echarts/src/constants.ts

```diff
@@ -18,3 +18,18 @@
  */
 // eslint-disable-next-line import/prefer-default-export
 export const NULL_STRING = '<NULL>';
+
+export const TIMESERIES_CONSTANTS = {
+  grid_offset_right: 40,
+  grid_offset_left: 20,
+  grid_offset_top: 20,
+  grid_offset_bottom: 20,
+  grid_offset_bottom_zoomable: 80,
+  legend_right_top_offset: 30,
+  legend_top_right_offset: 55,
+  zoom_bottom: 30,
+  toolbox_top: 0,
+  toolbox_right: 5,
+  data_zoom_start: 0,
+  data_zoom_end: 100,
+};
```

Comment:
Do you know which ones of these are globally relevant for ECharts charts and which ones only apply to the Timeseries chart? I'd prefer to split these so that the Timeseries-specific ones are under `src/Timeseries/constants.ts` and the globally relevant ones are under `src/constants.ts`.

Reply:
I think these constants are only for this chart, the Pie and BoxPlot does not have a zoom option, and legend fit ok